corehq/apps/export/tests/test_table_configuration.py

```diff
@@ -12,6 +13,24 @@
 )
 
 
+class TableConfigurationDuplicateTests(SimpleTestCase):
+    def test_table_containing_duplicate_paths_with_differing_doc_types_can_find_either(self):
+        path = [PathNode(name='closed')]
+        prop1 = ScalarItem(path=path)
+        prop2 = ExportItem(path=path)
+        table_config = TableConfiguration(
+            columns=[
+                ExportColumn(item=prop1), ExportColumn(item=prop2)
+            ]
+        )
+
+        scalarIndex, _ = table_config.get_column(path, 'ScalarItem', None)
+        exportIndex, _ = table_config.get_column(path, 'ExportItem', None)
+
+        self.assertEqual(scalarIndex, 0)
+        self.assertEqual(exportIndex, 1)
```

Comment:
After reading the description of this PR (and knowing the context from pairing with you on this bug), I expected this test to verify that there can only be a single column with a given path. However, it looks like this is verifying that the duplicate items, both having the same path, exist in separate columns. In other words, this test seems like it is ensuring that duplicates can exist when I expected it to verify the opposite. I'm confused. What did I miss?